### PR TITLE
JS: added tribonacci_numbers.js

### PR DESCRIPTION
**Pull request by ryachen01**
**Changes:**

Added file tribonacci_numbers.js for tribonacci numbers problem

### CODE REVIEW

File: code/mathematical_algorithms/src/tribonacci_numbers/tribonacci_numbers.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+function tribonacci(n) {
+  if (n == 0){
```
Comment (arnavb) [inline code/mathematical_algorithms/src/tribonacci_numbers/tribonacci_numbers.js]: Add spaces around all operator, spaces before braces, and remove the last else statement. The very last line is missing a semicolon, which is somehow in the comment.

**Comment by ryachen01** [comment]
@arnavb 
I fixed the issues as requested

**Comment by AdiChat** [comment]
Looks great 😍 

You have made a wonderful contribution. Join us [**at our community forum**](https://discourse.opengenus.org/signup) :family_man_woman_girl_boy: to get connected and access to community benefits

If you would like to [write an article on this topic](https://discourse.opengenus.org/t/become-a-writer-at-opengenus-iq/843), let us know and we will be happy to have you on board

Hope you are enjoying your journey 🚆  with **OpenGenus**
positive_opengenus

File: code/mathematical_algorithms/src/tribonacci_numbers/tribonacci_numbers.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  if (n == 0) {
+    return 0;
+  }
+  else if (n == 1) {
+    return 0;
+  }
+  else if (n == 2) {
+    return 1;
+  }
+  return tribonacci(n-3) + tribonacci(n-2) + tribonacci(n-1);
```
Comment (arnavb) [inline code/mathematical_algorithms/src/tribonacci_numbers/tribonacci_numbers.js]: One more minor change before this is good to merge: Add spaces around all operators please.

Comment (ryachen01) [reply code/mathematical_algorithms/src/tribonacci_numbers/tribonacci_numbers.js]: @AdiChat @arnavb 

I fixed this. It should be good to merge now

**Pull request merged**